Tests/ReactiveSwiftTests/PropertySpec.swift

```diff
@@ -265,6 +265,39 @@ class PropertySpec: QuickSpec {
 				property = nil
 				expect(isEnded) == true
 			}
+
+			it("should not deadlock") {
+				let queue: DispatchQueue
+
+				if #available(macOS 10.10, *) {
+					queue = DispatchQueue.global(qos: .userInitiated)
```

Comment:
I think that if you want to match "high priority" you should consider using the `.userInteractive` QoS instead. Doubt it really matters in a synthetic benchmark like this, but the asymmetry stood out to me. :)